independent-publisher-connectors/haveibeenpwned/apiDefinition.swagger.json

```diff
@@ -0,0 +1,484 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "haveibeenpwned",

```

Comment:
@troystaylor Can you please change it to "Have I Been Pwned" as the company or service website also uses this wording.